public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -119,59 +117,54 @@ figure.image-display
   :marked
     ## Constructor or variable initialization?
 
-    We initialized our component properties using variable assignment.
-    This is a wonderfully concise and compact technique.
-
-    Some folks prefer to declare the properties and initialize them within a constructor like this:
+    You initialized your component properties using variable assignment. You could have
+    declared the properties and initialized them within a constructor like this:
   +makeExcerpt('app/app-ctor.component.ts', 'class')
 
   :marked
-    That's fine too. The choice is a matter of taste and organization policy.
-    We'll adopt the more terse "variable assignment" style in this chapter simply because
-    there will be less code to read.
+    This page demonstrates the more terse "variable assignment" style simply for
```

Comment:
Ward mentioned using "chapter" instead of "page" because "page" can be confused with the app. This is the first time I've noticed that problem, although I don't think it matters too much in this case which one you mean. Maybe:

This app uses the more terse...